admin_submit_quest.py

```diff
@@ -33,10 +34,18 @@ class Quest(db.Model):
     test_outputs = db.Column(db.Text, nullable=True)
     xp = db.Column(db.Enum('30', '60', '100', name='xp_points'), nullable=False)
     type = db.Column(db.String(20), nullable=True)
+    is_active = db.Column(db.Boolean, default=True, nullable=True)
 
     def __repr__(self):
         return f"QuestID={self.quest_id}, Quest Name='{self.quest_name}', Language='{self.language}', Difficulty='{self.difficulty}', XP='{self.xp}'"
     
+
+class ReportedQuest(db.Model):
+    __tablename__ = 'reported_quests'
+    quest_id = db.Column(db.String(10), ForeignKey('coding_quests.quest_id'), primary_key=True)
+    report_status = db.Column(db.Enum('In Progress', 'Resolved', 'Not Resolved', name='repost_status'), nullable=False)
+
```

Comment:
class:ReportedQuest has very few attributes, only quest_id and the current status would not be enough. In the `reported_quests` table we should have the entire info for the quest like:

* who is the author
* who reported the quest
* who created it
* when it was created originally
* when it was reported
* when it was fixed(eventually)
* name, language, difficulty
* text field(maybe JSON) to store the history of the modifications that has been done
and more...

Reply:
These are the new fields we might create:
![image](https://github.com/SoftUni-Discord-Community/skill_forge/assets/122923696/d0ba9f0c-39d5-4158-b39b-c3fbd895ec8e)

To do:
- Check the type of the field "lastly_fixed" (I'm not sure if it should be like this)
- Add new field that will store the history of modifications (I'm not sure how to do this and how the functionality will work)
- Make the migrations --> `important!!!!!`

**I will not commit these changes for now.**